contrib/great_expectations_semantic_types_expectations/great_expectations_semantic_types_expectations/expectations/expect_column_values_to_be_secure_passwords.py

```diff
@@ -1,33 +1,14 @@
-import json
-import re  # regular expressions
 from typing import Optional
 
-from great_expectations import *
+from great_expectations import *  # noqa: F403
```

Comment:
Any reason we wouldn't want to drop this as well?

Reply:
Done